test/e2e-test/regression-test/util/interceptors.js

```diff
@@ -0,0 +1,123 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+const _ = require('lodash');
+
+class Interceptor {
+    constructor (next) {
+        this.__next = next;
+    }
+
+    handle (data) {
+        if (!!this.__next && this.__next instanceof Interceptor) {
+            return this.__next.handle(data);
+        }
+        return data;
+    }
+}
+
+class DoNothingInterceptor extends Interceptor {
+    handle (data) {
+        return super.handle(data);
+    }
+}
+
+class ExtraDynamicFieldInterceptor extends Interceptor {
+    constructor (next) {
+        super(next);
+        this.__placeholder = 'removed';
+        this.__removeUUIDResourceTypes = [
+            'DocumentReference', 'Composition', 'Immunization', 'MedicationStatement',
+            'Condition', 'Observation'
+        ];
+    }
+
+    handle (data) {
+        if (!_.isPlainObject(data)) {
+            return data;
+        }
+        data = this.__handle(data);
+        return super.handle(data);
+    }
+
+    __handle (data) {
+        if (!('entry' in data) || !_.isArray(data['entry'])) {
+            return data;
+        }
+        const entries = data['entry'];
+
+        for (const entry of entries) {
+            if (!entry || !('resource' in entry) || !('resourceType' in entry['resource'])) {
+                continue;
+            }
+
+            const resource = entry['resource'];
+            this.__removeDocumentReference(resource);
+            this.__removeSectionReferences(resource);
+            this.__removeResourceIds(resource, entry);
+        }
+        return data;
+    }
+
+    __removeDocumentReference(resource) {
+        if (resource['resourceType'] != 'DocumentReference') {
+            return;
+        }
+        resource['date'] = this.__placeholder;
+
+        // The zlib.gzip result will be different on different platforms, see https://stackoverflow.com/questions/26516369/zlib-gzip-produces-different-results-for-same-input-on-different-oses.
+        // Hence the hash result will be different too, which will trigger NodeJS CI error and need to be removed.
+
+        if (!('content' in resource) || !_.isArray(resource['content'])) {
+            return;
+        }
+        for (const ele of resource['content']) {
+            if ('attachment' in ele) {
+                if ('hash' in ele['attachment']) {
+                    ele['attachment']['hash'] = 'removed-hash';
+                }
+                if ('data' in ele['attachment']) {
+                    ele['attachment']['data'] = 'removed-data';
+                }
+            }
+        }
+    }
+
+    __removeSectionReferences(resource) {
+        if (!('section' in resource)) {
```

Comment:
Could you please give an example of this section field?

Reply:
Using sample data `170.314B2_Amb_CCD.cda` and template `ccd.hbs`, we will generate the following resource which contains a `section` field.

```json
{
  "resourceType": "Bundle",
  "type": "batch",
  "entry": [
    {
      "fullUrl": "urn:uuid:600552e7-9ea0-3324-a314-aa2772770e8c",
      "resource": {
        "resourceType": "Composition",
        "id": "600552e7-9ea0-3324-a314-aa2772770e8c",
        "identifier": {
          "use": "official",
          "value": "1.3.6.1.4.1.22812.3.99930.3.1"
        },
        "status": "final",
        "type": {
          "coding": [
            {
              "code": "34133-9",
              "display": "SUMMARIZATION OF EPISODE NOTE",
              "system": "http://loinc.org"
            }
          ]
        },
        "date": "2013-01-30T13:00:51.000Z",
        "title": "Continuity of Care Document",
        "confidentiality": "V",
        "attester": [
          {
            "mode": "legal",
            "time": "2013-01-30T13:00:51.000Z",
            "party": {
              "reference": "Practitioner/c479e34b-fd7d-3990-8450-1746cb2d2fc9"
            }
          }
        ],
        "event": [
          {
            "period": {
              "start": "2012-08-15T09:00:51.000Z"
            }
          }
        ],
        "section": [
          {
            "title": "Reason For Referral",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Reason For Referral</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "42349-1",
                  "display": "Reason For Referral",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot"
          },
          {
            "title": "Functional Status",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Functional Status</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "47420-5",
                  "display": "Functional Status",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "Condition/981a3a1b-bf1f-39be-bf90-7d63d530a79f"
              },
              {
                "reference": "Condition/27c44f66-c856-317b-b86a-a27b1144855b"
              }
            ]
          },
          {
            "title": "Problems",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Problems</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "11450-4",
                  "display": "Problem List",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "Condition/2e298bd2-c7a7-3e4c-9b04-62fdcc458048"
              },
              {
                "reference": "Condition/f504b238-7cb9-3e16-9f11-65704b6b0e21"
              }
            ]
          },
          {
            "title": "Medications",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Medications</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "10160-0",
                  "display": "History of medication use",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "MedicationStatement/15f2de74-09de-3158-bd6a-37a8ba545506"
              }
            ]
          },
          {
            "title": "Allergies and Adverse Reactions",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Allergies and Adverse Reactions</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "48765-2",
                  "display": "Allergies, adverse reactions, alerts",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "AllergyIntolerance/cdf8b57d-f863-3dd3-b933-c14016a9c203",
                "display": "Penicillin G benzathine"
              },
              {
                "reference": "AllergyIntolerance/529d06cc-640e-3493-9006-bd95f2a4214a",
                "display": "Codeine"
              }
            ]
          },
          {
            "title": "Immunization",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Immunization</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "11369-6",
                  "display": "Immunizations",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "Immunization/a1e4769a-d837-3f2c-9fe8-bdcc13057c6d"
              }
            ]
          },
          {
            "title": "Social History",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Social History</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "29762-2",
                  "display": "Social History",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "Observation/4cc0353b-0d2a-366c-ba89-e4052fd7d1ac",
                "display": "Assertion"
              },
              {
                "reference": "Observation/5622c980-8d5a-31f2-bc95-0f17975b5f27",
                "display": "Assertion"
              }
            ]
          },
          {
            "title": "Vital Signs",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Vital Signs</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "8716-3",
                  "display": "Vital Signs",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot"
          },
          {
            "title": "Results",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Results</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "30954-2",
                  "display": "Relevant diagnostic tests and/or laboratory data",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "DiagnosticReport/c5f3619c-0fb2-3ee6-b7a0-ad92376b0bc1"
              }
            ]
          },
          {
            "title": "Plan of Care",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Plan of Care</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "18776-5",
                  "display": "Plan of Care",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "CarePlan/db05ea6f-0775-3a67-9440-6d964ac5c701"
              },
              {
                "reference": "CarePlan/046ba3ec-c5a3-3eda-b2e3-3c15ecc86dbb"
              },
              {
                "reference": "CarePlan/2896304b-a4c6-3f98-854f-228422278eb8"
              }
            ]
          },
          {
            "title": "Instructions",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Instructions</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "69730-0",
                  "display": "Instructions",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot"
          },
          {
            "title": "Encounters",
            "text": {
              "status": "generated",
              "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Encounters</div>"
            },
            "code": {
              "coding": [
                {
                  "code": "46240-8",
                  "display": "Encounters",
                  "system": "http://loinc.org"
                }
              ]
            },
            "mode": "snapshot",
            "entry": [
              {
                "reference": "Encounter/3d1f600d-b0b9-36ec-881c-90d5e69d3715"
              }
            ]
          }
        ],
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        },
        "encounter": {
          "reference": "Encounter/def59397-9507-34b0-8f1b-0c01500e4903"
        },
        "custodian": {
          "reference": "Organization/f8d02634-0f53-3647-87f7-bb430a374a52"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Composition/600552e7-9ea0-3324-a314-aa2772770e8c"
      }
    },
    {
      "fullUrl": "urn:uuid:c479e34b-fd7d-3990-8450-1746cb2d2fc9",
      "resource": {
        "resourceType": "Practitioner",
        "meta": {
          "profile": [
            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner"
          ]
        },
        "id": "c479e34b-fd7d-3990-8450-1746cb2d2fc9"
      },
      "request": {
        "method": "PUT",
        "url": "Practitioner/c479e34b-fd7d-3990-8450-1746cb2d2fc9"
      }
    },
    {
      "fullUrl": "urn:uuid:def59397-9507-34b0-8f1b-0c01500e4903",
      "resource": {
        "resourceType": "Encounter",
        "id": "def59397-9507-34b0-8f1b-0c01500e4903",
        "status": "unknown",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.3.4",
            "value": "5283815"
          }
        ],
        "period": {
          "start": "2012-08-15T09:00:51.000Z",
          "end": "2013-01-30T13:00:51.000Z"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Encounter/def59397-9507-34b0-8f1b-0c01500e4903"
      }
    },
    {
      "fullUrl": "urn:uuid:f8d02634-0f53-3647-87f7-bb430a374a52",
      "resource": {
        "resourceType": "Organization",
        "id": "f8d02634-0f53-3647-87f7-bb430a374a52",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3",
            "value": "3"
          }
        ],
        "name": "Get Well Clinic",
        "address": [
          {
            "line": [
              "Get Well Clinic",
              "1004 Healthcare Dr."
            ],
            "city": "Portland",
            "state": "OR",
            "country": "US",
            "postalCode": "97005"
          }
        ],
        "telecom": [
          {
            "system": "phone",
            "value": "+1-(555)555-1004",
            "use": "temp"
          }
        ]
      },
      "request": {
        "method": "PUT",
        "url": "Organization/f8d02634-0f53-3647-87f7-bb430a374a52"
      }
    },
    {
      "fullUrl": "urn:uuid:92769d70-b4a6-393b-94e1-6af32a4c35d4",
      "resource": {
        "resourceType": "Patient",
        "meta": {
          "profile": [
            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient"
          ]
        },
        "id": "92769d70-b4a6-393b-94e1-6af32a4c35d4",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3",
            "value": "110107073916280"
          }
        ],
        "name": [
          {
            "family": "Everyman",
            "given": [
              "Adam"
            ]
          }
        ],
        "birthDate": "1962-10-22",
        "extension": [
          {
            "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
            "extension": [
              {
                "url": "ombCategory",
                "valueCoding": {
                  "code": "2106-3",
                  "display": "White",
                  "system": "urn:oid:2.16.840.1.113883.6.238"
                }
              },
              {
                "url": "text",
                "valueString": "White"
              }
            ]
          },
          {
            "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity",
            "extension": [
              {
                "url": "ombCategory",
                "valueCoding": {
                  "code": "2186-5",
                  "display": "Not Hispanic or Latino",
                  "system": "urn:oid:2.16.840.1.113883.6.238"
                }
              },
              {
                "url": "text",
                "valueString": "Not Hispanic or Latino"
              }
            ]
          }
        ],
        "address": [
          {
            "use": "home",
            "line": [
              "1234 Six Forks"
            ],
            "city": "Portland",
            "state": "OR",
            "country": "US",
            "postalCode": "97005"
          }
        ],
        "telecom": [
          {
            "system": "phone",
            "value": "+1-(555)555-3333",
            "use": "home"
          }
        ],
        "communication": [
          {
            "language": {
              "coding": [
                {
                  "system": "urn:ietf:bcp:47",
                  "code": "eng"
                }
              ]
            },
            "preferred": true
          }
        ]
      },
      "request": {
        "method": "PUT",
        "url": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
      }
    },
    {
      "fullUrl": "urn:uuid:7124235e-bb04-31bd-8022-123cef646356",
      "resource": {
        "resourceType": "AllergyIntolerance",
        "id": "7124235e-bb04-31bd-8022-123cef646356",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.10.2",
            "value": "613618700107"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.10.2",
            "value": "613618700109"
          }
        ],
        "clinicalStatus": {
          "coding": [
            {
              "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
              "code": "resolved"
            }
          ]
        },
        "reaction": [
          {
            "substance": {
              "coding": [
                {
                  "code": "7982",
                  "display": "Penicillin G benzathine",
                  "system": "http://www.nlm.nih.gov/research/umls/rxnorm"
                }
              ]
            },
            "manifestation": [
              {
                "coding": [
                  {
                    "code": "247472004",
                    "display": "Hives",
                    "system": "http://snomed.info/sct"
                  }
                ]
              }
            ]
          }
        ],
        "onsetDateTime": "1998-01-10",
        "patient": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "AllergyIntolerance/7124235e-bb04-31bd-8022-123cef646356"
      }
    },
    {
      "fullUrl": "urn:uuid:e0803d45-7917-3f1c-9f81-ae9717745cc9",
      "resource": {
        "resourceType": "AllergyIntolerance",
        "id": "e0803d45-7917-3f1c-9f81-ae9717745cc9",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.10.2",
            "value": "624373800001"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.10.2",
            "value": "637073500005"
          }
        ],
        "clinicalStatus": {
          "coding": [
            {
              "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
              "code": "resolved"
            }
          ]
        },
        "reaction": [
          {
            "substance": {
              "coding": [
                {
                  "code": "2670",
                  "display": "Codeine",
                  "system": "http://snomed.info/sct"
                }
              ]
            },
            "manifestation": [
              {
                "coding": [
                  {
                    "code": "422587007",
                    "display": "Nausea",
                    "system": "http://snomed.info/sct"
                  }
                ]
              }
            ]
          }
        ],
        "onsetDateTime": "2001-01-10T00:00:00.000Z",
        "patient": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "AllergyIntolerance/e0803d45-7917-3f1c-9f81-ae9717745cc9"
      }
    },
    {
      "fullUrl": "urn:uuid:9c0bf8bb-34da-344e-ba34-85bd0b777627",
      "resource": {
        "resourceType": "MedicationStatement",
        "id": "9c0bf8bb-34da-344e-ba34-85bd0b777627",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.9",
            "value": "659122500005"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.9",
            "value": "659122500007"
          }
        ],
        "status": "completed",
        "dosage": [
          {
            "doseAndRate": [
              {
                "doseQuantity": {
                  "value": 2,
                  "unit": "Puffs"
                }
              }
            ]
          }
        ],
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        },
        "medicationReference": {
          "reference": "Medication/79af1fe5-461e-3134-b395-e9c0b94078ce"
        }
      },
      "request": {
        "method": "PUT",
        "url": "MedicationStatement/9c0bf8bb-34da-344e-ba34-85bd0b777627"
      }
    },
    {
      "fullUrl": "urn:uuid:79af1fe5-461e-3134-b395-e9c0b94078ce",
      "resource": {
        "resourceType": "Medication",
        "id": "79af1fe5-461e-3134-b395-e9c0b94078ce",
        "code": {
          "coding": [
            {
              "code": "573621",
              "display": "Albuterol 0.09 MG/ACTUAT metered dose",
              "system": "http://www.nlm.nih.gov/research/umls/rxnorm"
            }
          ]
        }
      },
      "request": {
        "method": "PUT",
        "url": "Medication/79af1fe5-461e-3134-b395-e9c0b94078ce"
      }
    },
    {
      "fullUrl": "urn:uuid:e4183100-6a93-36fc-ae64-54c6466a5a56",
      "resource": {
        "resourceType": "Condition",
        "id": "e4183100-6a93-36fc-ae64-54c6466a5a56",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.1.2.1",
            "value": "615028800003"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.1.2.1",
            "value": "659935200001"
          }
        ],
        "code": {
          "coding": [
            {
              "code": "64109004",
              "display": "Costochondritis",
              "system": "http://snomed.info/sct"
            }
          ]
        },
        "onsetDateTime": "2012-08-15",
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Condition/e4183100-6a93-36fc-ae64-54c6466a5a56"
      }
    },
    {
      "fullUrl": "urn:uuid:7fb63c83-8033-3c16-95bd-86105c4b0a6b",
      "resource": {
        "resourceType": "Condition",
        "id": "7fb63c83-8033-3c16-95bd-86105c4b0a6b",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.1.2.1",
            "value": "636842200007"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.1.2.1",
            "value": "660100900011"
          }
        ],
        "code": {
          "coding": [
            {
              "code": "195967001",
              "display": "Asthma",
              "system": "http://snomed.info/sct"
            }
          ]
        },
        "onsetDateTime": "2011-09-25T05:00:00.000Z",
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Condition/7fb63c83-8033-3c16-95bd-86105c4b0a6b"
      }
    },
    {
      "fullUrl": "urn:uuid:bce14af7-2667-345f-8a93-c3916be5f145",
      "resource": {
        "resourceType": "DiagnosticReport",
        "id": "bce14af7-2667-345f-8a93-c3916be5f145",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.5",
            "value": "659988700005"
          }
        ],
        "status": "final",
        "code": {
          "coding": [
            {
              "code": "82374",
              "display": "CO2",
              "system": "urn:oid:2.16.840.1.113883.6.12"
            }
          ]
        },
        "effectiveDateTime": "2012-08-15",
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        },
        "performer": [
          {
            "reference": "Practitioner/undefined"
          }
        ],
        "result": [
          {
            "reference": "Observation/7622585b-002d-341c-8bea-cec41f537774"
          }
        ]
      },
      "request": {
        "method": "PUT",
        "url": "DiagnosticReport/bce14af7-2667-345f-8a93-c3916be5f145"
      }
    },
    {
      "fullUrl": "urn:uuid:7622585b-002d-341c-8bea-cec41f537774",
      "resource": {
        "resourceType": "Observation",
        "meta": {
          "profile": [
            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-observationresults"
          ]
        },
        "id": "7622585b-002d-341c-8bea-cec41f537774",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.5",
            "value": "6599887000031"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.5",
            "value": "6599889000031"
          }
        ],
        "category": [
          {
            "coding": [
              {
                "system": "http://terminology.hl7.org/CodeSystem/observation-category",
                "code": "laboratory"
              }
            ]
          }
        ],
        "status": "final",
        "code": {
          "coding": [
            {
              "code": "2028-9",
              "display": "CO2",
              "system": "http://loinc.org"
            }
          ]
        },
        "effectiveDateTime": "2012-08-15",
        "valueQuantity": {
          "value": 25,
          "unit": "mmol/L"
        },
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Observation/7622585b-002d-341c-8bea-cec41f537774"
      }
    },
    {
      "fullUrl": "urn:uuid:c8943173-2976-3803-a940-f44263fab4e5",
      "resource": {
        "resourceType": "Observation",
        "meta": {
          "profile": [
            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-observationresults"
          ]
        },
        "id": "c8943173-2976-3803-a940-f44263fab4e5",
        "category": [
          {
            "coding": [
              {
                "system": "http://terminology.hl7.org/CodeSystem/observation-category",
                "code": "social-history"
              }
            ]
          }
        ],
        "status": "final",
        "code": {
          "coding": [
            {
              "code": "ASSERTION",
              "display": "Assertion",
              "system": "urn:oid:2.16.840.1.113883.5.4"
            }
          ]
        },
        "effectivePeriod": {
          "start": "2012"
        },
        "valueCodeableConcept": {
          "coding": [
            {
              "code": "266919005",
              "display": "Never smoker",
              "system": "http://snomed.info/sct"
            }
          ]
        },
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Observation/c8943173-2976-3803-a940-f44263fab4e5"
      }
    },
    {
      "fullUrl": "urn:uuid:0dd56536-5b01-36fc-8030-3845a7f11873",
      "resource": {
        "resourceType": "Observation",
        "meta": {
          "profile": [
            "http://hl7.org/fhir/us/core/StructureDefinition/us-core-observationresults"
          ]
        },
        "id": "0dd56536-5b01-36fc-8030-3845a7f11873",
        "category": [
          {
            "coding": [
              {
                "system": "http://terminology.hl7.org/CodeSystem/observation-category",
                "code": "social-history"
              }
            ]
          }
        ],
        "status": "final",
        "code": {
          "coding": [
            {
              "code": "ASSERTION",
              "display": "Assertion",
              "system": "urn:oid:2.16.840.1.113883.5.4"
            }
          ]
        },
        "effectivePeriod": {
          "start": "2012"
        },
        "valueCodeableConcept": {
          "coding": [
            {
              "code": "428041000124106",
              "display": "Current some day smoker",
              "system": "http://snomed.info/sct"
            }
          ]
        },
        "subject": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Observation/0dd56536-5b01-36fc-8030-3845a7f11873"
      }
    },
    {
      "fullUrl": "urn:uuid:a1e4769a-d837-3f2c-9fe8-bdcc13057c6d",
      "resource": {
        "resourceType": "Immunization",
        "id": "a1e4769a-d837-3f2c-9fe8-bdcc13057c6d",
        "identifier": [
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.4",
            "value": "637382500009"
          },
          {
            "system": "urn:oid:1.3.6.1.4.1.22812.3.99930.3.4.4",
            "value": "637382500011"
          }
        ],
        "occurrenceDateTime": "2012-08-15",
        "vaccineCode": {
          "coding": [
            {
              "code": "88",
              "display": "Influenza virus vaccine",
              "system": "urn:oid:2.16.840.1.113883.12.292"
            }
          ]
        },
        "status": "completed",
        "route": {
          "coding": [
            {
              "code": "C28161",
              "display": "INTRAMUSCULAR",
              "system": "urn:oid:2.16.840.1.113883.3.26.1.1"
            }
          ]
        },
        "site": {
          "coding": [
            {
              "code": "368208006",
              "display": "Left Arm",
              "system": "http://snomed.info/sct"
            }
          ]
        },
        "patient": {
          "reference": "Patient/92769d70-b4a6-393b-94e1-6af32a4c35d4"
        }
      },
      "request": {
        "method": "PUT",
        "url": "Immunization/a1e4769a-d837-3f2c-9fe8-bdcc13057c6d"
      }
    },
    {
      "fullUrl": "urn:uuid:49027ad2-afbf-39ba-a3b7-cc6e8a0ba3cd",
      "resource": {
        "resourceType": "DocumentReference",
        "id": "49027ad2-afbf-39ba-a3b7-cc6e8a0ba3cd",
        "type": {
          "coding": [
            {
              "code": "34133-9",
              "display": "SUMMARIZATION OF EPISODE NOTE",
              "system": "http://loinc.org"
            }
          ]
        },
        "date": "2020-10-19T02:49:42.707Z",
        "status": "current",
        "content": [
          {
            "attachment": {
              "contentType": "text/plain",
              "data": "H4sIAAAAAAAAA+09aXPqyLWfnV/Rz+/DTOoF0AIC7uM6hQFfO/HCAL4zeanUlAwNqCIkSouvPb/+dWtD3VJLLSFj7OBJZmwt3erTZ1/69P76stFrtvOqQ3sNoQOc1y38+pMDX5zGi63/BNYWXH79ae042y+Nxo8fP+raGtZVXbfnlrZ17Lluuov63Nw0BqNhQ6pLjensH7ejKf6z/tv09qe/Xvyp91+1GugVmkgU6qKg1CWx3mmyRwa1Ghp9oGuGNlf1oTl3N9BwAJrIsL+eu5bxZa23a6a1+vIsn/uXv7zY2tfz2HJ+yHX0QEMSBLHx293tdL6GG7WmGbajGnN4fvGns54FVX0zMBcQzNG/vp4/Ts8b+Dpewc0CoBVAw9ZM4+v5+GEw/P16KKCfpnAOLNN0vp5LdVFBy0Arqoui3OnI6Bc5GAFutrrqUKPc3I1vfx8M+xPp99vR99GtmDGSkBiI/WhdQv+T8JUyL0n+SxrxqYqitIVuVwq/EK2srqCHm/gFqSNK6O9utysL6L/BrPMdIOWmKMu17rn35/TVRh+U9hFowPgj9+oGvXv7cHM/OAcLzUZrePWvTR/v7vqTm//rz24e7sHDFRiNb6YPwxG4f5iNgjVrjg4vBqbhaIarOa/AXIKBakEQYk+v4T+CnoXLJZw72jOcaRsInlXdRXNIgijj/wkdQWiJNaElCOG6jKW2QCNoqo4GjvCFva5WXWqdB6D4nlzh9W0bDJKD+rPpqrFy1RWMoSU0aiFmWnBuWouZaq2gg/4+621VR0OjTExvZWf0NoqiIAptoS13RUXqCBy76c2DxlEXCwu4Nv7cc+/KWc92LETffXQD2vatZsALUZKbYKq9gCvT+rfdaySf8N+coyVejE3LQatb9Bren+GYCE8vHib4Xfybf3Vror90DIKLbhttR68RuxIMabqGY71ePE7ReMHv3oc38Jf7vzpQR/DahDuM/vzyP2Lt51ar9Wf0/5qMfs6DNY7DdQcADWYx0Ib5v571lupG018vRs/Qet2oRq8RXAhurxA+GRf9hbrpNfzf/SEauzEQUDeIpSEoqRj7vkFjAa3YRt/lEUwroDU81pNmOes4AotdRRIFSYqesNQ55MBWxI3lToiuaASlJlPk9+tac2A0LHTWiC1/s0x3W3z0jlJrUaPfmw64RhdUNCowLXCLgGOY0XQ7gthsXCwPHITZIdA5OJ1c73TQ73VZqkegKfQi+jXrxTgtiV25jX9vYa5Yl3a7xaDr1e7+FklJaEEkmm6MRYSxlruDe4MNCUQfMbRFY5nPiLlYD9YKAfWPGMQo9iDzM4SAFr4hAf8r1HXgC2cCu3PITUSyMxosItJUvpKYhcFYUnkSmgZcI8nurOce/7fqGe8zORODNfHwJjZzIrhTg7VN0W4GXL3XoNh+T3WdtekNg+RevghDL9i2tjLgoh+96KECC/WbdeWcFCPhT8gmSTzq4vcKChemUOHd/D33/kiFUoxKol0bQ8uOKDhFJP19zRJHU3TNIuSRz2m0l4uhhTHQ+5UWVOhzEzPvrkU4hC5Fv1aFkQWZE/7nhFSlkMoHvGashvBZm4eTIcXGXapzx7WgdYe+Qccy+mJkONDaWpoNweh6AkLDDPhiHyAhicyselPC1l1bxqtIHyeAhLl0fiBIlhr6f8HMUg17aVob4IncuthBII0PSaJw2jqRHo1Q30YsFi5OQvKohWTGTuWwpLlrO+ZCUw2C3wziVwlEiO6cMOKjYARzy3aoEd9vNGr8L83AXMR7ZwLn2jZUoNENB5tmC+JqSfEknsRTIfGEVF2ItJU0tnyipMopKR3WvUYaBeCrqQTT0+FK1TEPxv4sZJYWUgQxmapYT4hZxlNKSURKQgiXhIMrskwyrZkTEX58wyPCgl4jFeM8I+SEgicUPAgK0sjWWwSBBo8/Pix9zIIWNjpGzzh8NdfRMAMPu8aD8eQ8DZlaUkfuiC0+3aITAmvO44cVpWQoYjCe1Zqhd7bbleRIXSFiJAGUdPNHjJgkAX2o0MXEFEnVtbZa59IbAl5y9N4WWli6QMsLIAZgmlyFwYdCfted9l3MQdws8x4m7+i9pWvM8f5zeMbRZjabcgj88Zhyi48tbaNar34Qaxy4Cnffp+0oAW1L5EbGGxA5jHcPpbGxPRkZN9oJKUG+cD0YTHYUHVAE/+c3ChT9zZOrYzJ7DRicD5Qrda7pmqOhEaJPiql1e+l1eyt2WZodS7Xj0u0ylLs4ry2mDqfzW5LjZvLcLK7L5rsE503nvancF1+M+MVRso83IE8/XMnteDiR50cnT7Eked4jJHM0A2nIkEWl96oxf01Qqe0uMWlO7tG6/V8/OZUeWsgPaCE/MA3b1fFmHUbES9HPpxXxotBsk2wAIfnHZgTNbjlGMEZEDC0WDxhDw0jygLeW1AiSMZvIs6SShhMC3WZrGuhaYEdBA1/BoxurkeEBE6YGDQvZT813MnbQJcZ6MKLEV74DhG9OOpbrxdQWl+biNbSpq0k1PKMm86zXeWayDZs7y/HUl1hiYFOSm92aWFli4ASqCPtw+hmY4PwZS9VjqTpevl/KI7tUwGBdL06MaLHr0wE3QwTEoTDq94f9Qf8cEUQd+PQEtjoaEgIbQuCsNRsE2RoAsQ8AMT4gGIGnVxDgvcfcXX1jGticuwpkB5hB27HxfnvzRXS1+xhMKfNYig+5O4U3KyMHFN1qpm9XuykJtVZl2xUuHsd3Ebm7Nr1ZiQeytkrX7OgP9Nnq1lt/YohQoNzYtgsxxIMHozc19M3RX2gO9Unf8VJ8YQ3VRewCumLF/8RPXOAFoo9dJ24M0SbY6Xego2p64h76Oz48vk3O33OeQupnftACeBnZvob1slE145x4YofosecuTX1BPZYkiMt+/3LkkcS9CWLAvkH8TLP8TFsSqaOFpF9O+ZCXhfbM/JD4cwjpTWOB6CpCpBKTqfbNyjAtmJjxyeIaFe3QIhP8C4wCHPDnXrauPkH9/OLBsKGTs6QvDEiwIJSBGGKr1ndXNSwNC21xPnx8QqgSQrbPXqhHM6Hpc4svgAWY9Jc9+U9jHxP9uOCTZAAkvaMLBIvqNeIcrNeI80UGkxyYK0PDKsmJR74Rj+xHPHIH6xOLZEL/xCJPLLIAfCpkkXGVGxuaOyv9rGc+YWPVs0vj0buHy+k52Jjmwv9z9P1+B8VCinezrnRigKGdNYLQ5jFkm1StQVyHz1Lau0pSa5/eP9yNhmAwizw2zU5LVgRBYavyurbRfOM9/glxVf0M5/UE5Q2hxfzfhDIZe7FBvNnz0TQWmceGjw4duIi/lGLBe9dTrHhirvT3et4r4MXWvnjVjOeDYQgPRWm12oIg59tEOeClamBiwiFGkj3T0lYaAvKMgGYBcKI1po7Ra3gv7kgihusRZcSp4eCk0ZaPnDTktoxmQT8Jp24o8ENHMVhqxkKLlfsUoI/+iT6qpA8anFXRxyG9NS2Gd00Umy0hzOeowF0ztkwkRDfgFklL2lMT3Mt00FAymNLRSYWcYR+wrAOmbUBoBrRdQFsF9CfkWAR89kDCGhhcXv7dMwYGpu2Y8zVSxy3EHxLOv+CT0y5y6XlF1H/+adJVf1rxTxuPUtryVf49lrm/us+4WlrVLwORFCV/n61PU/D3UO8zlPud5XsVInvfExsFYJ0PMFLrPwzxIokRrMd21hv1RLMfmWalVm0Kt5hmxRPN7mh2GOH4G9MsZaeTVnqWIa7OifTZPlYQq7EyMowMRWwJUqeDNXyuYp8maWhQg7XQUy0JDyaWGCym5Q0esKKWreK1YjkcldgIhqvrV7r6bFpfzx/v/85jI3g7OIG6pzLba20by/mZPl7+jVDji1uThXe6SXDT/feaMiyr2PHkkNVZq62WIrSTfpxAjydnJW1TpjkVKrYNkvopU4wL+djol2ukslEw21Btijifg8NaKmKoUio+KRYYxmoh+LLs1YTFiq9kp9sFD6XmjaVidNEE8XCCWMJWhBPlUz+ZQxRLAA2Gycj+iiZKzQELQcyRCRZOxcwHCzcwAalCOdzUfpIZYsFNIk8suJaV1R08kpHbHcKBlTcWri0xc3oOGX0vgZV0Tmm46t05EhHsyOIvP4VLxPy4JjSj/CwabP3EQB4ZsAWg/zY+Lot+Jg9jcwb1kTP3IQ8rc5/aYWX+rD5u5i05A1vzgJWHoflTc2BOYiNjhwNEl7KUlMnoakIx8DJqSmFFJcFAPSWAPBDPNxP2cPQFLmwZaR+1BAuh5T9TQkXmPr1BiRF41YAMmV2BCtRSRE/BI3OPPc0hIWUT/l7/egJj4rZO2ksZryCMnO/ikBnRlvexf2Sl05Q8BZY32JIxmIJZbxcrwyf75wjtn+J7nWv/FN/xD2b/hL7Bw9o/yGyRmHpMKVNI7La6SlsQOPKyi9hCvsd0PxMoDcSFTKB8vTATnie9MDbrSS886YWceuEwjW5PeuG+euFBa0JYWQaCqCBOWVmWwbVme3V/5hJs4CI41hYfy0ynHNxFdz981sEuvPg3L/ASn6JEHJMvuOTFwKKZh8HUff3JdaBl6kCoC11w962BrIvH/gxtBroMF2Bh2ukhoUPEI6sJoyYEGRnjvQ1AMfv1AYwfr66m4BflGown96A/BcObyWgwGw3re4bFPiRkLkPI/DDB1l0ubQDx8efA1l7A2nQtG6g2MCBcIDxZmhb4sYbwD81YVRpCZEShG1lPpEVlcdIvGUJFF65Ma4YZBgU1nC7M+WzAcYinvSu/as5aM9LeCFhCoVcSBQzBuhPXMsPFuN7DQrI1Fi+WhNoVfAoSWZLjUanWwSW6uuIskXLtXyPrRs6o6hTm16aFwTOWldCLWDHvrAj3oWLZtvvk9yDpx7oBkIrj9LI/vIurjjf3s9IeHlHJ8Mq0uqKErB/0w1Uz3ax3uQbj9CEQg3ElxPr8qNJs2JhGefP99vfZNO6YoUOPkkTYh34NeBnnEOMLxsEnANyoRnNcjBnTLZxrSw0uvp4vVR3pRcDcQgsfloB01PjHoquaGTUPQJqva2hoD9Y8H2SZSAuI4MZW6+S6pHiqYUK3ux/cgNka2qprudGBDwO5I4m0z2Uymo5vJv3Zw+Qf4Oeb++v+rdfY5c/x78Raxy+u6kV/oh0IV4QLseMCgGisYRqP6CFqKcxP5F0pWkpT6ip0rnd/NHmYPtz+BdyNZqPJaEj6LA3b3XiyYWc8DqZ38S2LnZO9GFvmwiUdvXf9+8c9XH6STErJ+Gx3aBRLU3WKF0eE02rLisThEep0kqgwebk3rQ0NKh5Fk2L5THcRiz8MaQbhIz1rHMzzyTitHyHFh5cgrq3pibglcbR5ChDJ+8GWxshvhxQxZpDnKUbcAKldmMveGIugLUhsyuzoACE7iqMQ5V+MYUhTkltKM9Wl6p384vEwfGAFXpWnHjoWVB0vhX9Pw72Mt/YyiRplnbVx+zzb9i/iSHrrKIBYb7VpyHG/2mwz8SB0FlXsimoU35cUH9Ho0D6iEs4ehkb43g4ghXWIS6ettGpSZQ6gvq5Da6VB+y9AXWA+BwHiEz7bQJfQTSdxUEj0jnfqSj94bRK+9tH9Q/s6hDgS24eXt55pP4aGNtd0XTPAN/AEjT9UbIKWNZHeNmM8Wdr8VrXfBdKpg3IQofY31aiJ3W7njZ0f1Sach5RWAK4hleUnne/cSIMA264Rt0wvfDo2T1pBF0ROIj5PHv7wWwCk6pwUB+Ill/3boMBtAU+8oyzvwK1qT7wjiVv9ALvuVdeG6RVYJ+ZxOepXzTyOsVpHyHA/irIidnAaC3cKE99o3TKjHYu+zGs78ZwQgJUbnC9TjQf0yNLjKPt2f9yKdY2tCL/oEWM41p9ORxPsRM3PjGyWs6y50tPSMKRMThqPuY4IS1QENJ8gUQSysNwVogNMJa9Zpj2xiqKZaYH9dKrNOdXmnGpzolsla3P8s5MFPwezdcrBPOVgYrColqPNta2KNF9WAI18DDfMzgqf4aeRlPAO3fYCi/Fn7/qzpCETScV2t8OhwXEHwxhet0SeCDsAVkgwnWXIprOzlDBYyI6Y3sEUlkGAltx3ao9IlrG7R+gb+wXFymqNhfVGqUNTQFpopCLVnzo1O5hwjxTag4dHzsql0Gbjw93VdHYs+NClYb63JSESqbhJJCut+58VyegOXbhVZnQzbYqzvKqrAGGY72dUu0jNdrMtVV7573m2aQGSwb6LwTmLfSeNiwOlqRcz4mHY5yJOfCPM4tgmfAXNPaJFxLuG5CzsGOoys3xTUlNuy50iJ4tkjSa3hbZcJAfw83u6sCse+zH8n/9Af1cJDMvxd5XAsuP3d7Hx5JN6vYJI3xuWY55cASdXwLG6AuLrfZhdV2GtpxHUWba1joP6qmH7kiRdxVXaHPV5xc618iL7Sa2U5TiIUgFOjoKTo+DkKCjhKCiugh2Jo6CfxtI+oKOgKUkt7KlpV3yWvZfGUoWnIBXQR+kpOIqCdol5bL6sdGtKZbz3ZrNxDe0PHwh04nL85kfPUt5Vsd+8XxW7P/WNsUSIbfyhgmfNcm1EKPN5kUzET3COdlg84NVSIeQ6yInax38KflT4rDkwN8Euduo7jdI549/CpQP61ubI0hPft4DZgCvvlqdnBbWr1bhHW3HXUNLVJHf8OmTO1Komz2C8p5gRg2EoeseaqDNPu8B/N142etzbx6h0vqm+0jmj089b1f8i8CgiLSDvZ5P+3eN08HjbnxBlvNutZarzNabV2MpkpSPhxtocIrqIIhaSbPl+QiGbIMu/0tsJ0Wvb3Xr/OuEW5SMtUidMpjykToRUr66U3JfB999oxEiX4vsXBdO7xN6osxSnxlvU/Oan/7GT/ygWJaMfDtZEbjHtFdw77a+SpL+8lL+shD/udL+cZL+ExzQ/0U+k6CfD75l0hwVZfffaau1oxkrVYWqmX5Dnd48E72tanh8Sy0uc2je5xwWlSyrLL6WOnfWRzPy+tOy+zxsBPtKiXLGdasNK3bYiVRh5nZpzxOJAcDYbbcSSd7PM2PT209ON+W+E6kHz6fx20xl69z0+kArYaEBopSjAXGchpVsv1OFFBb5p4FqWd8tEOtdCfT3A5x1RS9s20dI23f1InROLPsmrI+NwSyawtz/3znp5ozQAkQhBlw7+v9nRlZKidMWuF1MnnYwxsiA0bCq0ljgMVxQQH8VBesKRzA675gVd80OuRQOuHOFWnmArd6iVK9DKVhqygZOtKORNmSvBqe1KhFY/r/g+vnbGJ7Z4OLbYlDpCE7MxUUL/oQ8oYsjnInxSwoeGn/jkiU9+cj55SNNmtxDCv9QWlRpHO3FOw+a7hpAZTNGuJ0JzsVtVRuZwCC4lAjeDtpNyeQJtV0+7UXHE7jonYocjVJhZk6REPeTAWGPwtAcsbzV8Z/U2GMsivvuG/8MRyv2wtyr24XPGknKDWQeNEXrAzZvej8JdQ+w8KjFHuD9807SF3RRAM6qK4x3+sIyrAI32CZ8RaHmdi5YFdvPXw+ym2G3FtlN/+rjbGYK/su38VuV2Xo4BFkqmjl3ob72nrTiJbjb/vF796wPv67eK9/Wq4n0dauqhNlb4VBt7VfHG5uszRTYWH81/hzR3cGMs4AsYqPrcxWrs4s23WUo9PPADb/Ro/40unadiWivV0P4g7bDB7eN0NppU5dWRsg7XVxRBws5ZgSs+Sx3PQtTVKUrHL5yqMv/Bt3dswhQ6qzyxpMJAd5kzbk6B7ioD3UIFgW7mWTZ5J9lkn2PzZhHufFcgwnhRkNP6B578gZ/UH0g5vrxrh6jalbLLdiOJI2FBUFTkZEmwVrnxIiG2TzfCjix4uQ3keQuelUx5lvg6AaRY/xW1bc0QhGfJcMf4l/PwBVyv5/c2+adm/O7a/6Lwna+7PYMLZfGhfE7Ew4uKcyMufsTHkQrwJE6ulMKXODlTJm/i4U5Z/CmDQ6XwKGaVTpJ3HSk7E8U4QxM7FXC0+JByq+SQlTA1WWyKtS7F1H4tz9RSfGDvzdTEbiviavrT7+ozwdU+HXpy1gcUws9yY1YjdZsdoUaHzmPOzFJYmuLRe3csbUWy1/erfWIklarnoSWHrAhFFSnReCjuly2FoynOyXfHUeE/B0dlzqKtAjhacshq5HxXbCk1Oj+T7WMuhbIpbtb3RlnMF3yMFStE1uPLrUl8JS4sChzPwaV3y6iRGRXvstBtNSusFpggy+gZn82z0NSVgWwxbQ5wwMXrzoXAAXT1yfQ6lr6CheqodOaNn/RyiKybIbTnlrYNqu/pu98x6laYe0OERiKsOGd/NyOVhXKr7VkK7UPbKxJPrwJ/5/blsTQcIApfZLlETMy0FtDCyMkXeNsFrIb9b34ro4fUHKBqo30tADYbU2/c7hH1y0qbIl/PT9TyAPbW+VzVxvMu+7PZaPKPquJ5cVmVbG/d7eDzYLhPMCTiYHzKRFqZ7DhKa+5IcptWdxGifozo3alMNQDpqUz18GWqvEE8+RTEOwXx3twazoy57cSMzOn9a+WP191jvErMYEmQOgl3Nym7Cpi7gZZ2VPZua+ehwSodtSfYatxaSDXCmIW/qYxNfESmZewMcOIstU677Xk7KrIsx0jAAnMJBkgw01Zj/F7xGnT8NmIS4GEHeo5KdNIMTRqitNp9xjqcbWel0sezZZictIqfMDtDwzP7g3LOZuO1MbNMG/K55AFbw/7lZd8ztL6Zqv4F+IEvcGvajO7ADCuSdZkyntJt60S7hj2WvWscy+psuscpauzF8xvlSbO82KjF4MA1Uf4mpWS2VrNNGOv2QiiKEI+Tygb9PkFlfdtZb1RwpxrqCm7QkydaO9Hah6M1yoF0RrmQ8N/Mo1HytIFR6OEvpQscg+iN1UTgH0z8Y1ffmIaKrLila3g6H8DVpyxJ+9Hw/yLaNDCBqp3RXDz1oM9cMIGfLW/YP1cAr4IrG1p1MIaG8QrGLrJSLPCwXGrIGKoBA2IHvv+X6nsPjnY732bUrWU+a2le/rwXblXb93RfBFDdzfqXANpDi1X08rFkXe0KPmFRlxpCOYpdLAgGQfgSr4R7X2GSFazgCVT3J7+UDlQ0Mzp6taRmh6u5IBngzjzkYjK6mux7yEXBFXaz3Fol1xj3VhQ/UScSNHh9+zVSiqQzFSlltlKak+EB/syQlHmSvjLObleiLKB/pXS7yj0DCE/KbIhVcS90Kfqpvhe6KDTbZLAIrefztzJvJrqVlG1lHorcrGbmnhA+0mbm6a2obh8G3K2opkP66aKsUZbo3Yi7YCVRqnG0f5JatDu+jzSf/gqCO+0FKZf9uYtjTY+G5tCzcVZCMBtsjW8HtIrk7VpBdfddW0tNpg/3lAg4SNsgMadvULvj/8PZM4gKuPPJR95T2ISmgt4TElNwNxOKmWd0kcwHaieUHWEt14Cn4DFf9OMVHIb47eG2vPaa0XGh1epInPXyokzEUitEXanTFZWukDg6cPQCrblm48OLHRPopg3BD7pmh0Jv5qH0QQQk9mJeiwcD/mBm8cQmnENPMc9I62EdyHgq1v9E6T7/ecX6b3Ts65tyOt7mfm/G6pqCIiqS12WQOn+WChYVZnFR+Ol4WNxxHXi5++C4Aq9027LAo8Hz9qMzEHdz5+nt6GL3+HMoGEfhR2HHy4HnUE9gEEDc1XStOfQKA4AWmxwE3tkFWLgWFq/qYqPZtpenT7v1OE/m30VB/c9ZaEh04HmhL8QxvF3DhjqebhN0dUl8xzOS9ilx0pJuQZXs29PHZLhjNDf3s9KMRhIKsgdRxO9263KKr2uL6AN9+Gjhzj1Kwc2rItUIGY5KO3nk/BTqyxqWqGPq5eJ8w8efAlwjRdNntpz8hFsid5AkaSdZuK/ljC1zY5bbiP5bbsRBe39Kqay2qUhNoZbbwYqb1e7CxzSjjQeW+aucEgf4EgrggXpsXvWHiHtiVwwE39O5YbB/71g0lLekZKx46NeqafkH3BHBYgyLAdLgzfnaNBaW5mjpgde94ZGYfRhOTzossk7689tqv0lPykJQTxhtR4VAVY/HjkzzxqV9s2sXlfbMLFZU+h16zTKj0eXzrj4I/xC6rAD0oY61LBx53SfuWlXxdCXlb92uZy4TEpcsqCnYv9OTbGQifnrzTiL4yqc7UWPTmhNnmxVs1QrdRJuRsg1XTv69k3/Pv/Ih/Xt80ddCsdf9Iq+HjLvyFdVxx1x9nEhQW2LfGBFVdjw1PZpaKLko2yTfL62oIzB3UEKSq5M4TyjFYtjDXiQrz/i7Cp0dKr5MMpWkBtBqihJvbFki6ulJcFcQopOkruj1eCf2a7dLe0Wdh4cNOSdKkWmZf8YW6949Vi81PKbckuTK26dTRjBtlrK7dBcDOzsnDt/y3qcgzc9o/A8tR1aFCYvONwuJgXSi+ob7/qdOyYgOazQtpxIBez+G6RuSRglFaCEDWfdmCkgzUkSccUJJWI9oUjArNfXiLDv5gv1iTs6GSnSFL5utkek29TRd129Ef+mbl8QTvYYv6FV9aM7djXft/wF5TeP/KyYBAA==",
              "hash": "MUcL9gFpcpUCaPKTzUfl4JOLQok="
            }
          }
        ]
      },
      "request": {
        "method": "PUT",
        "url": "DocumentReference/49027ad2-afbf-39ba-a3b7-cc6e8a0ba3cd"
      }
    }
  ]
}
```